core/src/integration-test/java/com/scalar/db/util/AdminTestUtils.java

```diff
@@ -0,0 +1,41 @@
+package com.scalar.db.util;
+
+import com.scalar.db.config.DatabaseConfig;
+import java.util.Properties;
+
+/** Utility class used to delete or truncate the metadata table */
+public abstract class AdminTestUtils {
```

Comment:
This adds utility method to delete and truncate the metadata table to be used in integration testing.